tests/test_integration.py

```diff
@@ -0,0 +1,520 @@
+import os
+import pathlib
+import re
+import subprocess
+import sys
+from typing import Any
+from typing import Dict
+from typing import Optional
+from typing import Sequence
+from typing import Union
+
+import pytest
+
+from riot.riot import _T_CompletedProcess
+
+
+_T_Path = Union[str, "os.PathLike[Any]"]
+
+
+def run(
+    args: Union[str, Sequence[str]], cwd: _T_Path, env: Optional[Dict[str, str]] = None
+) -> _T_CompletedProcess:
+    if isinstance(args, str):
+        args = args.split(" ")
+
+    return subprocess.run(
+        args,
+        env=env,
+        encoding=sys.getdefaultencoding(),
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        cwd=cwd,
+    )
+
+
+def test_no_riotfile(tmp_path: pathlib.Path) -> None:
+    result = run("riot", cwd=tmp_path)
+    assert (
+        result.stdout
+        == """
+Usage: riot [OPTIONS] COMMAND [ARGS]...
+
+Options:
+  -f, --file PATH  [default: riotfile.py]
+  -v, --verbose
+  -d, --debug
+  --version        Show the version and exit.
+  --help           Show this message and exit.
+
+Commands:
+  generate  Generate base virtual environments.
+  list      List all virtual env instances matching a pattern.
+  run       Run virtualenv instances with names matching a pattern.
+""".lstrip()
+    )
+    assert result.stderr == ""
+    assert result.returncode == 0
+
+    result = run("riot list", cwd=tmp_path)
```

Comment:
we should get the same for `riot run` and `riot generate` right?

Reply:
Correct, are you suggesting that we should also check those here? Would be an easy parametrization